Member-Travel-Fund.md

```diff
@@ -0,0 +1,40 @@
+## Purpose
+
+To establish and administer a fund for members of the Node.js
+Foundation to travel and spread knowledge about the foundation
+and the Node.js Foundation projects.
+
+## Restrictions
+
+* Candidates must be Individual Members of the Node.js Foundation.
+* Candidates must be presenting one of the following presentations:
+ * [The Node.js Foundation](https://github.com/nodejs/foundation-slides).
+
+## Process
+
+New slide decks are added to the list of approved decks on an ongoing basis.
+
+* Slide deck must be managed by the Node.js Evangelism WG.
+* Send a pull request to this file adding the slides to
+ the list above.
+* Slide deck must be approved by the TSC.
+
+Any member can apply to give an approved talk. Request may take up to a month
+for the TSC to approve or decline.
+
+* Log an issue in this repository.
+ * Include the name of the event you plan to attend.
+ * Include the location of the event and where you will be traveling from.
+ * Include the presentation you intend to give.
+ * Include the size of the stipend you wish to receive.
+  * Stipends are anywhere from $100 to $1,500 USD and vary with travel distance.
+
+The TSC will consider each proposal in its next regular meeting. While it is
+ultimately at the TSC's discretion who to approve the following considerations
+are made.
+
+* Impact: Preference given to underserved communities.
```

Comment:
The thinking here is that within a subject we're trying to promote awareness about (right now that's just general knowledge about the foundation and the project but we expect that to change over time) there are people, communities and geographies where that knowledge is not very wide spread. For instance, a meeting in San Francisco would be less attractive than a meetup of the same size in Cincinnati. This is especially true when we start looking at non-english speaking countries and non-JS conferences (FOSDEM, Grace Hopper, etc).

This is a bit vague because our understanding of under-served areas is going to evolve over time, as is the scope of the information we intend to persist will grow as we add more slide decks.